button-merchant/src/main/java/com/usebutton/merchant/Order.java

```diff
@@ -25,16 +25,30 @@
 
 package com.usebutton.merchant;
 
-import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.Date;
+import java.util.List;
 
 /**
  * Represents an order placed by the user to be tracked using ButtonMerchant.trackOrder(order).
  */
 public class Order {
-    @NonNull
+
     private String id;
+    @Deprecated
     private long amount;
     private String currencyCode;
+    @Nullable
+    private Date purchaseDate;
+    @Nullable
+    private List<LineItem> lineItems;
```

Comment:
Why is this nullable?

Reply:
Even though it's required there's no default value